packages/gestalt/src/Tile/Tile.js

```diff
@@ -0,0 +1,161 @@
+  /**
+   * Ties a visible checkbox to the tile's selected state
+   */
+  showCheckbox?: boolean,
+  /**
+   * Adds a Tooltip on hover/focus of the Tile. See the With Tooltip variant to learn more.
+   */
+  tooltip?: string,
+|};
+
+function DisabledOverlay() {
+  return <div className={classnames(styles.tile, styles.disabledOverlay)} />;
+}
+
+function TooltipWrapper({
+  tooltip,
+  disabled,
+  children,
+}: {|
+  tooltip?: string,
+  children: Node,
+  disabled?: boolean,
+|}) {
+  const tooltipZIndex = new FixedZIndex(1);
+  if (!tooltip || disabled) return children;
+  return (
+    <Tooltip idealDirection="up" text={tooltip} zIndex={tooltipZIndex}>
+      {children}
+    </Tooltip>
+  );
+}
+
+/**
+ * [Tile] https://gestalt.pinterest.systems/web/tiledata component should be used for ... on the page.
+ */
+export default function Tile({
+  bgColor,
+  borderColor,
+  id = '',
+  disabled = false,
+  children,
+  selected = false,
+  tooltip,
+  showCheckbox,
+  onChange,
+}: Props): Node {
+  const { handleOnBlur, handleOnMouseEnter, handleOnMouseLeave, isHovered } =
+    useInteractiveStates();
+
+  const [isSelected, setIsSelected] = useState(selected);
+  const { isFocusVisible } = useFocusVisible();
+
+  useEffect(() => {
+    setIsSelected(selected);
+  }, [selected]);
+
+  const classes = classnames(styles.tile, {
+    [styles.selected]: isSelected,
+    [styles.focused]: isFocusVisible,
+    [styles.hovered]: isHovered && !isFocusVisible,
+    [styles.disabled]: disabled,
+  });
+
+  const handleClick = () => {
+    setIsSelected(!isSelected);
+    if (onChange) {
+      onChange({ id, selected: !isSelected });
+    }
+  };
+
+  const generateSelectedColorStyles = () => {
+    const colorStyles: {| borderColor: string, backgroundColor: string |} = {};
+    if (!isSelected) return styles;
+
+    // the interal base component uses hex codes
+    // but could be passed in pre-tokenized values
+    if (borderColor && borderColor.startsWith('#')) {
+      colorStyles.borderColor = `${borderColor}`;
+    }
+    if (bgColor && bgColor.startsWith('#')) {
+      colorStyles.backgroundColor = `${bgColor}`;
+    }
+    return colorStyles;
+  };
+
+  const handleKeyDown = (event: SyntheticKeyboardEvent<HTMLDivElement>) => {
+    if (event.keyCode === ENTER) {
+      setIsSelected(!isSelected);
+    }
+  };
+
+  return (
+    <div style={{ position: 'relative' }}>
+      {disabled && <DisabledOverlay />}
+      <TooltipWrapper tooltip={tooltip}>
+        <div
+          tabIndex={disabled ? null : 0}
+          role="button"
+          aria-pressed={selected}
+          className={classes}
+          onBlur={handleOnBlur}
+          onClick={handleClick}
+          onMouseEnter={handleOnMouseEnter}
+          onMouseLeave={handleOnMouseLeave}
+          onKeyDown={handleKeyDown}
+          style={generateSelectedColorStyles()}
+        >
+          <Flex direction="row" gap={2}>
+            <Flex.Item>{children}</Flex.Item>
```

Comment:
Since you're not applying any styles or grouping multiple child components, these `Flex.Item`s don't seem necessary (they're applied automatically under the hood)